app/src/androidTest/java/com/aws/amazonlocation/ui/main/GeofenceEditTest.kt

```diff
@@ -135,21 +148,32 @@ class GeofenceEditTest : BaseTestMainActivity() {
             ),
         )
 
-        Thread.sleep(DELAY_1000)
-
         val seekbar = mActivityRule.activity.findViewById<SeekBar>(R.id.seekbar_geofence_radius)
         seekbar.progress = ((seekbar.max - seekbar.min) * getRandom0_01To1_0()).toInt()
-        Thread.sleep(DELAY_1000)
+
         updatedRadius = seekbar.progress
 
         onView(withId(R.id.btn_add_geofence_save)).perform(click())
 
-        uiDevice.wait(Until.hasObject(By.res("${BuildConfig.APPLICATION_ID}:id/rv_geofence")), DELAY_5000)
+        uiDevice.wait(
+            Until.hasObject(By.res("${BuildConfig.APPLICATION_ID}:id/rv_geofence")),
+            DELAY_5000,
+        )
     }
 
     private fun verifyGeoFenceUpdate() {
-        Thread.sleep(DELAY_1000)
-        onView(withId(R.id.rv_geofence)).perform(RecyclerViewActions.scrollTo<RecyclerView.ViewHolder>(hasDescendant(withText(geofenceName))))
+        waitForView(
+            CoreMatchers.allOf(
+                withId(R.id.rv_geofence),
+                isDisplayed(),
+                hasMinimumChildCount(1),
+            ),
+        )
+        onView(withId(R.id.rv_geofence)).perform(
+            RecyclerViewActions.scrollTo<RecyclerView.ViewHolder>(
+                hasDescendant(withText(geofenceName)),
+            ),
+        )
```

Comment:
minor: Thanks for the all the indentation/styling fixes! It would be nice to see if we can add static analysis (like FindBugs) on this package to flag these as potential warnings. That way these are fixed when the code is initially committed.